docs/concepts/internationalization.md

```diff
@@ -37,6 +37,29 @@ By default, `#getDescriptionId` will return `block.` or `item.` prepended to the
 The only purpose of a translation key is internationalization. Do not use them for logic. Use registry names instead.
 :::
 
+Mod metadata
+------------
+
+A mod's name and description has historically been defined in [`mods.toml` files][modinfo] without translation support, however since Neoforge v20.4.179 translation files can override mod info using the following keys:
+
+| Translation Key | Overrides | Example |
+|:---------------:|:---------:|:--------|
+| `fml.menu.mods.info.displayname.[modid]` | `[[mods]]` -> `displayName` | `fml.menu.mods.info.displayname.examplemod` |
+| `fml.menu.mods.info.description.[modid]` | `[[mods]]` -> `description` | `fml.menu.mods.info.description.examplemod` |
+
+For example, a mod with the id `mymod` could localize its name and description by adding the following lines to a language file:
+
+```js
+{
+  "fml.menu.mods.info.displayname.mymod": "My Mod",
+  "fml.menu.mods.info.description.mymod": "My Mod is super cool and localized!"
+}
+```
+
+:::note
+It is still recommended to define `displayName` and `description` in your `mods.toml` file, because the localization keys are not garenteed to be supported everywhere. This is especially true for third-party tools.
```

Comment:
```suggestion
It is still recommended to define `displayName` and `description` in your `mods.toml` file, because the localization keys are not guaranteed to be supported everywhere. This is especially true for third-party tools.
```